app/models/page.rb

```diff
@@ -68,4 +77,8 @@ def verified_teacher_permissions?
   def public_permissions?
     viewer_permissions == "Public"
   end
+
+  def display_category
+    category || "Uncategorized"
+  end
```

Comment:
Use 
.present?
.presence

bug because empty string is truthy.

Should actually use
`category.presence || "Uncategorized"`